src/registrar/tests/test_models_domain.py

```diff
@@ -532,9 +535,44 @@ def setUp(self):
             Given the registrant is logged in
             And the registrant is the admin on a domain
         """
-        pass
+        super().setUp()
+        self.nameserver1 = "ns1.my-nameserver-1.com"
+        self.nameserver2 = "ns1.my-nameserver-2.com"
+        self.nameserver3 = "ns1.cats-are-superior3.com"
+
+        self.domain, _ = Domain.objects.get_or_create(
+            name="my-nameserver.gov", state=Domain.State.DNS_NEEDED
+        )
+
+    def test_get_nameserver_changes(self):
```

Comment:
**comment**: There's lots of different types of changes loaded into this one test, changing IPs, taking away IPs, etc. I'd love to see them in separate tests for each kind of change. I know it's kind of a drag, but it's a nice way to make sure that we're testing all the mutations one by one, and then maybe a big test like this that does a lot of things to make sure they work together.